### PR TITLE
docs(modal): decompose Options demo

### LINKED ISSUE #3583: docs(modal): decompose Options demo
Decompose Options demo (related to #3258) to: 

- Disable animation 
- Disable Esc
- Disable backdrop & backdrop click

### PR DESCRIPTION
**Pull request by artolshansky**
Decompose option demo to: custom css class, animation, esc button and backdrop options

close #3583

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+modal/demos/service-options/custom-css-class/custom-css-class.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<button type="button" class="btn btn-primary" (click)="openModal(template)">Open modal</button>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+modal/demos/service-options/custom-css-class/custom-css-class.html]: why do we need a regular modal in the demo that shows a modal with css class?

File: demo/src/app/components/+modal/demos/service-options/disable-backdrop/disable-backdrop.ts [hunk 2]
```diff
@@ -3,14 +3,12 @@ import { BsModalService } from 'ngx-bootstrap/modal';
 import { BsModalRef } from 'ngx-bootstrap/modal/bs-modal-ref.service';
 
 @Component({
-  selector: 'demo-modal-service-options',
-  templateUrl: './service-options.html'
+  selector: 'demo-modal-service-disable-esc-closing',
```
Comment (IlyaSurmay) [inline demo/src/app/components/+modal/demos/service-options/disable-backdrop/disable-backdrop.ts]: wrong selector name

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+modal/demos/service-options/custom-css-class/custom-css-class.html (hunk 1 above)
Comment (artolshansky) [reply demo/src/app/components/+modal/demos/service-options/custom-css-class/custom-css-class.html]: yeap, don't needed

**Review by IlyaSurmay — APPROVED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — APPROVED** [review]

**Pull request merged**